dali/pipeline/operator/operator_test.cc

```diff
@@ -90,12 +90,26 @@ TYPED_TEST(OperatorDiagnosticsTest, DiagnosticsTest) {
   ASSERT_EQ(this->value_, cnt);
 }
 
+
 TYPED_TEST(OperatorDiagnosticsTest, DiagnosticsCollisionTest) {
   (this->operator_)->RegisterDiagnostic(this->value_name_, &this->value_);
   EXPECT_THROW((this->operator_)->RegisterDiagnostic(this->value_name_, &this->value_),
                std::runtime_error);
 }
 
 
+TYPED_TEST(OperatorDiagnosticsTest, IncorrectTypeTest) {
+  (this->operator_)->RegisterDiagnostic(this->value_name_, &this->value_);
+  EXPECT_THROW(this->operator_->template GetDiagnostic<long>(this->value_name_),  // NOLINT (long)
```

Comment:
```suggestion
  EXPECT_THROW(this->operator_->template GetDiagnostic<int64_t>(this->value_name_),  // NOLINT (long)
```
and get rid of the lint problems

Reply:
I just fancy using long here ;)

Reply:
Seriously, why long? It's asking for trouble - I even think it would be plain wrong if we tried to compile it with MSVC (I know we can't for other reasons, but `long == int32_t` is quite possible).

Reply:
That's an argument **for** using `long` :p
Cause this way this function won't throw, so the test will fail, suggesting that something is not yes.

Changed to `int64_t`